include/git2/remote.h

```diff
@@ -456,7 +468,7 @@ struct git_remote_callbacks {
 	 * not `NULL`, the update was rejected by the remote server
 	 * and `status` contains the reason given.
 	 */
```

Comment:
This comment is duplicated now. You may replace it with something like "See documentation of `git_push_update_reference_cb`".